SAMtags.tex

```diff
@@ -453,6 +455,112 @@ \subsubsection{Color space}
 Color read quality on the original strand of the read. Same encoding as {\sf QUAL}; same length as {\tt CS}.
 \end{description}
 
+\subsection{Base modifications}
+
+Base modifications, including base methylation, are represented as a
+series of edits from the primary unmodified sequence stored in the
+main SAM {\sf SEQ} field.  If the modifed base has no natural
+unmodified form then this should be stored as ``N''.
+
+Each modified base listed also has a quality value associated with
+it.  Given the unmodified base already has a phred likelihood, this
+base modification quality should be interpreted as the likelihood of
+this modification being correct, rather than the base being unmodified.
+
+\begin{description}
+\item[MM:Z:\tagregex{([ACGTN][-+][a-z](,[0-9]+)*;)*}]
+\hfill\\
+The first character is the unmodified base as seen in the {\sf SEQ}
```

Comment:
"As seen in the SEQ field" is confusing...if the read is rec-comp'ed the first character is the unmodified base s seen in the reverse-complement of SEQ...right?